admin/app/components/certification-centers/invitations-action.js

```diff
@@ -22,6 +26,25 @@ export default class CertificationCenterInvitationsAction extends Component {
     ];
   }
 
+  get rolesOptions() {
+    return [
+      {
+        label: this.intl.t('common.roles.admin'),
+        value: 'ADMIN',
+      },
+      { label: this.intl.t('common.roles.member'), value: 'MEMBER' },
+    ];
+  }
```

Comment:
Je suggère la modification ci-dessous, car : 
* la valeur de `rolesOptions` ne change jamais et donc il ne sert à rien de la réévaluer à chaque utilisation avec une fonction, l'intérêt d'un _getter_ étant d'être réévalué à chaque appel
* la propriété la plus importante de `rolesOptions` est sa `value`, sur laquelle tous les traitements se basent, et non son `label` et donc il est plus logique de la mettre en premier. La propriété `label` est annexe.

```suggestion
  rolesOptions = [
      {
        value: 'ADMIN',
        label: this.intl.t('common.roles.admin'),
      },
      { 
        value: 'MEMBER'
        label: this.intl.t('common.roles.member'), 
       },
    ];
```

Et logiquement il faudrait faire alors la même chose pour `languagesOptions` qui est déjà présente dans le code avant cette PR.

Reply:
Je ne suis pas très sur de comprendre pour le getter, la valeur change puisqu'on peut choisir entre Admin ou Member ?

Et pour mettre la value avant le label je ne suis pas très d'accord, je trouve ça étrange de lire la valeur de quelque chose avant même de savoir à quoi elle correspond.

Reply:
Ce que propose @lego-technix c'est d'utiliser un attribut de classe au lieu d'un getter. L'attribut aura toujours la même référence vers le tableau, le getter va retourner un nouveau tableau à chaque appel (si je ne me trompes pas).

Reply:
C'est plus clair et @lego-technix me l'a expliqué plus en détail en visio 🙂